### PR TITLE
Fix Help Logo & FireFox bug

### PR DESCRIPTION
**Pull request by SzeyinLee**
Updated font file & help logo

For Firefox
- need to use event.event.target instead of event.event.srcElement.
- fix centering issue

### CODE REVIEW

File: Viewer/src/viewer/defaultViewer.ts [hunk 1]
```diff
@@ -85,7 +85,7 @@ export class DefaultViewer extends AbstractViewer {
 
         let pointerDown = <PointerEvent>event.event;
         if (pointerDown.button !== 0) return;
-        var element = (<HTMLElement>event.event.srcElement);
+        var element = (<HTMLElement>event.event.target);
```
Comment (deltakosh) [inline Viewer/src/viewer/defaultViewer.ts]: Was it tested on edge and chrome?

Comment (SzeyinLee) [reply Viewer/src/viewer/defaultViewer.ts]: Yes

Comment (deltakosh) [reply Viewer/src/viewer/defaultViewer.ts]: You rock

Comment (SzeyinLee) [reply Viewer/src/viewer/defaultViewer.ts]: Should check with @RaananW if this is an issue.

**Pull request merged**